lib/elixir/lib/exception.ex

```diff
@@ -767,6 +772,7 @@ defmodule Exception do
       ""
 
   """
+  @spec format_file_line_column(term, term, term, String.t()) :: String.t()
```

Comment:
```suggestion
  @spec format_file_line_column(String.t | nil, non_neg_integer | nil, non_neg_integer | nil, String.t()) :: String.t()
```

Similar changes necessary above. :)